{"ID": "d8e125e3-bfbb-40c5-adcf-02622c955fd2", "effective_cve": "CVE-2015-20107", "image": {"exact": "docker.io/anchore/test_images@sha256:ff80ae5de3446939639f3fbb58f66f641e2da1d881bdab704d9237424f64417f"}, "label": "TP", "package": {"name": "python3", "version": "3.8.10-r0"}, "timestamp": "2023-02-13T18:00:03+00:00", "tool": "grype@v0.56.0-30-g17e11ac", "user": "westonsteimel", "vulnerability_id": "CVE-2015-20107"}